backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5569170_sys_gh10045_AddSysConfig.sql

```diff
@@ -0,0 +1,11 @@
+-- 2020-10-01T08:04:59.066Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) VALUES (0,0,541342,'C',TO_TIMESTAMP('2020-10-01 11:04:58','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','de.metas.handlingunits.impl.HUPIItemProductDAO.Enforce_ M_HU_PI_Item_Product_ID',TO_TIMESTAMP('2020-10-01 11:04:58','YYYY-MM-DD HH24:MI:SS'),100,'N')
+;
+
+
+-- 2020-10-01T10:34:07.379Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_SysConfig SET Name='de.metas.handlingunits.impl.HUPIItemProductDAO.Enforce_M_HU_PI_Item_Product_ID',Updated=TO_TIMESTAMP('2020-10-01 13:34:07','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_SysConfig_ID=541342
```

Comment:
rename it to `de.metas.ui.web.quickinput.field.PackingItemProductFieldHelper.EnforcePrecisePricePerHUItemProduct`